src/main/java/com/online/booking/service/impl/VehicleServiceImpl.java

```diff
@@ -0,0 +1,53 @@
+package com.online.booking.service.impl;
+
+import com.online.booking.models.VehicleModel;
+import com.online.booking.service.VehicleService;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import java.util.List;
+
+public class VehicleServiceImpl  implements VehicleService {
+    private Map<Integer, VehicleModel> vehicleDatabase = new HashMap<>();
```

Comment:
Vehicle details are present in database ? whats the point of using map ? Dont copy code blindly from anywhere